src/flag_gems/testing/__init__.py

```diff
@@ -7,6 +7,7 @@
     torch.float16: 1e-3,
     torch.float32: 1.3e-6,
     torch.bfloat16: 0.016,
+    torch.float64: 1e-12,
```

Comment:
we set the tolerance values of different data types according to pytorch/torch/testing/_comparison.py. it's recommended to choose 1e-7 for torch.float64.